tools/go-cli/go-whisk-cli/commands/package.go

```diff
@@ -42,17 +41,8 @@ var packageBindCmd = &cobra.Command{
   RunE: func(cmd *cobra.Command, args []string) error {
     var err error
 
-    if len(args) < 2 {
-      whisk.Debug(whisk.DbgError, "Package bind command must have at least one argument\n")
```

Comment:
Message is `at least one argument`, but the conditional check is `len(args) > 2` (at least **two** arguments)